src/coreclr/jit/emitxarch.cpp

```diff
@@ -11444,8 +11444,12 @@ BYTE* emitter::emitOutputAM(BYTE* dst, instrDesc* id, code_t code, CnsVal* addc)
             {
                 case IF_AWR_RRD_RRD:
                 {
-                    reg345 = id->idReg2();
-                    break;
+                    if (ins != INS_extractps)
```

Comment:
This seems like an interesting change. Why is this check here?

Reply:
There is some existing path that can go down `IF_AWR_RRD_RRD` for `extractps`, even though it should always go down `IF_AWR_RRD_CNS`.

That needs to be addressed separately as part of the larger existing issue covering invalid/nonsensical formats.

Reply:
- The change involved to properly fix this is a bit more complex/involved and seeing as this is an existing issue, I didn't feel the need to complicate this PR with that.

I plan fixing it in a followup PR alongside a couple other "obvious" places where the format is incorrect.